packages/expo-manifests/ios/EXManifests/EXManifestsNewManifest.swift

```diff
@@ -0,0 +1,78 @@
+//  Copyright © 2021 650 Industries. All rights reserved.
+
+import Foundation
+
+@objcMembers
+public class EXManifestsNewManifest: EXManifestsManifest {
```

Comment:
I think that it is preferable to not have long Objective-C names for classes and methods in Swift, and instead use the `@objc` annotation to specify the long name that it will have when called from Objective-C.

Example: https://github.com/expo/expo/blob/main/packages/expo-updates/ios/EXUpdates/Logging/UpdatesLogger.swift#L11

Reply:
If it's okay with you, I'll just do the rename in a follow-up (all of them at once). We won't have any objective-c soon so it's probably not worth worrying too much about the annotations or names (can do a programmatic rename last).